sufficient_statistics.cc

```diff
@@ -27,14 +27,18 @@ double SufficientStatistics::GetPopulationMutationRateStatistic(const TrioModel
   double GGGG = root_mat(128) / sum;
   double TTTT = root_mat(192) / sum;
 
-  // PrintReadDataVector(data.read_data_vec);
-  // cout << params.population_mutation_rate() << endl;
-  // cout << "Sum:\t"  << sum << endl;
-  // cout << "AAAA:\t" << AAAA << endl;
-  // cout << "CCCC:\t" << CCCC << endl;
-  // cout << "GGGG:\t" << GGGG << endl;
-  // cout << "TTTT:\t" << TTTT << endl;
 
+    if (sum == 0){
+        //Note: I can't reproduce this error
```

Comment:
I fixed it when I set the new theta in params in the EM. Before it was producing 0 here, because the theta was not updating after iterations.